core-bundle/src/Filesystem/Dbafs/Dbafs.php

```diff
@@ -537,7 +537,7 @@ private function applyChangeSet(ChangeSet $changeSet, array $allUuidsByPath): vo
                 return null;
             }
 
-            return $allUuidsByPath[$parentPath];
+            return $allUuidsByPath[$parentPath] ?? null;
```

Comment:
This does not look like the right fix to me. Why is there a path not in the map of "all uuids by path" in the first place?

Reply:
Would need to debug it further.
This happens when moving folders and in this case the $parentPath is the folder itself, however it does not exist in the set right now

Reply:
And it only happens in #8548 🤔 maybe @aschempp might know but gonna close this for now